lib/tests/it/main.rs

```diff
@@ -361,6 +371,47 @@ async fn test_container_import_derive() -> Result<()> {
     Ok(())
 }
 
+#[ignore]
```

Comment:
For the future, one approach could be to put those tests behind feature flags, see https://github.com/camallo/dkregistry-rs/blob/854d0da53bef5dd85b5e901123e85d43af97c74e/tests/net/quay_io/mod.rs#L16.

Reply:
OK, hmm yeah could make sense.